pkgs/development/haskell-modules/configuration-ghc-8.2.x.nix

```diff
@@ -71,4 +71,12 @@ self: super: {
   # https://github.com/aristidb/aws/issues/238
   aws = doJailbreak super.aws;
 
+  # There's no 8.2 testsuite
```

Comment:
The changes to `configuration-ghc-8.2.x.nix` seem to be orthogonal to the addition of the HIE package set. Can you please submit those fixes separately (or commit them directly) to take them out of this PR for easier reviewing?

Reply:
👍